pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/dao/CenterProfileDao.java

```diff
@@ -0,0 +1,17 @@
+package org.broadinstitute.ddp.db.dao;
+
+import org.broadinstitute.ddp.db.dto.CenterProfileDto;
+import org.jdbi.v3.sqlobject.SqlObject;
+import org.jdbi.v3.sqlobject.config.RegisterConstructorMapper;
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+import org.jdbi.v3.stringtemplate4.UseStringTemplateSqlLocator;
+
+import java.util.Optional;
+
+public interface CenterProfileDao extends SqlObject {
+    @UseStringTemplateSqlLocator
+    @SqlQuery("SELECT * FROM `center_profile` WHERE center_id = :id")
+    @RegisterConstructorMapper(CenterProfileDto.class)
+    Optional<CenterProfileDto> findById(@Bind("id") final Long id);
+}
```

Comment:
The Center-related code is out-of-scope for this ticket since none of the management or study-related configuration code is in place. Leave this for future expansion, but it's not currently used.